addons/binding/org.openhab.binding.groheondus/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,25 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: ., lib/ondus-api-0.0.2.jar
+Bundle-ManifestVersion: 2
+Bundle-Name: GROHE ONDUS Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.groheondus;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Import-Package: 
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.library.unit,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.eclipse.smarthome.core.util,
+ org.osgi.framework,
+ javax.measure,
+ org.slf4j
```

Comment:
Can you sort this import list alphabetically for some extra neatness? :-)

Reply:
Of course ;)